CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ PrestaShop 1.6.x, 1.7.x and 8.x
 
 # Changelog
 
+## [3.7.4]
+### Fixed
+- Fix: Improve exception handling and error logging.
+- Fix: Round off unit price in order line to 3 decimal digits.
```

Comment:
We discussed that 3 decimal points might not be supported

https://altapay.slack.com/archives/C0151NH0W1H/p1710327394852199?thread_ts=1710229402.184849&cid=C0151NH0W1H